scala-package/infer/src/main/scala/ml/dmlc/mxnet/infer/package.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ml.dmlc.mxnet
+
+import ml.dmlc.mxnet.infer.MXNetHandlerType.MXNetHandlerType
+
+package object infer {
+  private[mxnet] val handlerType: MXNetHandlerType = MXNetHandlerType.SingleThreadHandler
```

Comment:
can we change this variable anywhere?

Reply:
should we make it user-configurable?

Reply:
I will expose this as a property if I find that MXNet can be called from multiple threads.
, like said in other places I am going with the assumption that MXNet needs to be called from the same thread through out the lifetime of the process otherwise it seg-faults.